Runtime/Editor/LootLockerExtensionResponses.cs

```diff
@@ -0,0 +1,122 @@
+using System.Collections.Generic;
+
+namespace LootLocker.Extension.Requests
+{
+    //Login
+
+
+    public class MfaAdminLoginRequest
+    {
+        public string mfa_key { get; set; }
+        public string secret { get; set; }
+    }
+
+    public class AdminLoginRequest
+    {
+        public string email { get; set; }
+        public string password { get; set; }
+    }
+
+
+    //Games
+    [System.Serializable]
+    public class Game
+    {
+        public int id { get; set; }
+        public bool is_demo { get; set; }
+        public string name { get; set; }
+        public string badge_url { get; set; }
+        public string logo_url { get; set; }
+        public Game development { get; set; }
+    }
+
+    //Organization
+    [System.Serializable]
+    public class Organisation
+    {
+        public int id { get; set; }
+        public string name { get; set; }
+        public Game[] games { get; set; }
+
+        public Game GetGameByID(int id)
+        {
+            foreach (var game in games)
+            {
+                if (game.id == id)
+                {
+                    return game;
+                }
+            }
+            return null;
+        }
+    }
+
+    //User
+    [System.Serializable]
+    public class User
+    {
+        public int id { get; set; }
+        public string name { get; set; }
+        public string email { get; set; }
+        public long signed_up { get; set; }
+        public  Organisation[] organisations { get; set; }
+        private Dictionary<int, int> organisationIndexByID { get; set; }
+
+        public Organisation GetOrganisationByID(int id)
+        {
+            foreach (var org in organisations)
+            {
+                if (org.id == id)
+                {
+                    return org;
+                }
+            }
+            return null;
+        }
+    }
+
+    public class LoginResponse : LootLockerResponse
+    {
+
+        public string mfa_key { get; set; }
+        public string auth_token { get; set; }
+        public User user { get; set; }
+    }
+
+    //Roles
+    public class Permissions
+    {
+        public string permission { get; set; }
+    }
+    public class UserRoleResponse : LootLockerResponse
+    {
+        public string[] permissions { get; set; }
+        public bool self { get; set; }
+    }
+
+    //API Keys
+
+    public class Key : LootLockerResponse
```

Comment:
I don't think this needs to be a LootLockerResponse as I think it's only ever used in lists, no?